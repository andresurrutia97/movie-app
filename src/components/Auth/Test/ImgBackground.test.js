import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';

import ImgBackground from '../ImgBackground';

describe('<ImgBackground/>', () => {
  const component = <ImgBackground img="/imgPath" />;

  it('should render correctly ', () => {
    const tree = renderer.create(component).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('should contain an element with a brackground src that includes the path given', () => {
    const wrapper = shallow(component);
    expect(wrapper.props().style.backgroundImage).toMatch(
      new RegExp('/imgPath')
    );
  });
});
