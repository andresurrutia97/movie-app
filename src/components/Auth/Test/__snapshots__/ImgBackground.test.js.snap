// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ImgBackground/> should render correctly  1`] = `
<div
  className="w-screen h-screen bg-cover img-flip opacity-5  "
  style={
    Object {
      "backgroundImage": "url('/imgPath')",
    }
  }
/>
`;
